app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -266,11 +272,69 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector,
     fun setCheckedNavDrawerItem(@IdRes item: Int) = view_navigation.setCheckedItem(item)
 
     override fun showProgress() {
-        progressDialog = ProgressDialog.show(this, getString(R.string.app_name), getString(R.string.msg_log_out), true, false)
+        progressDialog = ProgressDialog.show(
+            this,
+            getString(R.string.app_name),
+            getString(R.string.msg_log_out),
+            true,
+            false
+        )
     }
 
     override fun hideProgress() {
         progressDialog?.dismiss()
         progressDialog = null
     }
-}
+
+    fun changeLanguage() {
+        val languages = arrayOf("English", "Hindi", "Japanese", "Russian", "Italian")
```

Comment:
Better to add this strings on the strings.xml file. We should also add the translations for all of them. Right now, this is hard-coded and only in English.